frontend/src/components/IssueV1/components/IssueTableV1.vue

```diff
@@ -0,0 +1,467 @@
+
+type Mode = "ALL" | "PROJECT";
+
+const columnList: BBTableColumn[] = [
+  {
+    title: "",
+  },
+  {
+    title: "",
+  },
+  {
+    title: "issue.table.name",
+  },
+  {
+    title: "issue.table.environment",
+  },
+  {
+    title: "issue.table.progress",
+  },
+  {
+    title: "issue.table.updated",
+  },
+  {
+    title: "issue.table.approver",
+  },
+  {
+    title: "issue.table.assignee",
+  },
+  {
+    title: "issue.table.creator",
+  },
+];
+
+interface LocalState {
+  dataSource: any[];
+  selectedIssueIdList: Set<string>;
+}
+
+const props = defineProps({
+  title: {
+    type: String,
+    required: true,
+  },
+  issueList: {
+    type: Array as PropType<ComposedIssue[]>,
+    default: () => [],
+  },
+  mode: {
+    default: "ALL",
+    type: String as PropType<Mode>,
+  },
+  leftBordered: {
+    default: true,
+    type: Boolean,
+  },
+  rightBordered: {
+    default: true,
+    type: Boolean,
+  },
+  topBordered: {
+    default: true,
+    type: Boolean,
+  },
+  bottomBordered: {
+    default: true,
+    type: Boolean,
+  },
+  highlightText: {
+    default: "",
+    required: false,
+    type: String,
+  },
+});
+const router = useRouter();
+
+const state = reactive<LocalState>({
+  dataSource: [],
+  selectedIssueIdList: new Set(),
+});
+const currentUserV1 = useCurrentUserV1();
+const environmentStore = useEnvironmentV1Store();
+
+const tableRef = ref<HTMLTableElement>();
+const isTableInViewport = useElementVisibilityInScrollParent(tableRef);
+
+const issueSectionList = computed(
+  (): BBTableSectionDataSource<ComposedIssue>[] => {
+    return [
+      {
+        title: props.title,
+        list: props.issueList,
+      },
+    ];
+  }
+);
+
+const selectedIssueList = computed(() => {
+  return props.issueList.filter((issue) =>
+    state.selectedIssueIdList.has(issue.uid)
+  );
+});
+
+const issueTaskStatus = (issue: ComposedIssue) => {
+  // For grant request issue, we always show the status as "NOT_STARTED" as task status.
+  if (!isDatabaseRelatedIssue(issue)) {
+    return Task_Status.NOT_STARTED;
+  }
+
+  return activeTaskInRollout(issue.rolloutEntity).status;
+};
+
+const activeEnvironmentForIssue = (issue: ComposedIssue) => {
+  const environmentName = activeEnvironmentInRollout(issue.rolloutEntity);
+  return environmentStore.getEnvironmentByName(environmentName);
+};
+
+const taskStepList = function (issue: ComposedIssue): BBStep[] {
+  if (!isDatabaseRelatedIssue(issue)) {
+    return [
+      {
+        status:
+          issue.status === IssueStatus.OPEN
+            ? "PENDING_APPROVAL_ACTIVE"
+            : (issueStatusToJSON(issue.status) as BBStepStatus),
+        payload: undefined,
+      },
+    ];
+  }
+
+  const res = issue.rolloutEntity.stages.map((stage) => {
+    const task = activeTaskInStageV1(stage);
+    let status = task_StatusToJSON(task.status) as BBStepStatus;
+    if (status == "PENDING" || status == "NOT_STARTED") {
```

Comment:
I'm pretty not sure about this... @LiuJi-Jim PTAL

Reply:
LGTM